### ISSUE TITLE
Multi-gpu training

### ISSUE THREAD

**Issue by jsaric** [sniklaus/softmax-splatting#9]
Hi Simon,
thank you for your time and effort.

So, I plugged your implementation of forward warping in my model 
and everything seems to work fine on single GPU setting.

However, when I try multi-gpu training the loss does not decrease,
and evaluation results are bad.

If I use classic  backward warping from native pytorch,
everything works fine.

Are you aware of any similar problems?
What do you think is causing the problem?

**Comment by sniklaus** [comment]
Thank you for your interest in our work! I have not heard of such issues before, could it be that you increase your batchsize and that you need to adjust your learning rate (I wouldn't be surprised if backward warping is less picky in terms of learning rate)?

**Comment by sniklaus** [comment]
Closing due to inactivity, please feel free to reopen in case this issue still persists. Thanks!
